### PR TITLE
Feat: Add applications to startpage

### PR DESCRIPTION
**Pull request by hblitza**
## Description

This reorders elements on the startpage:
- display all applications (public or for the specific user if logged in). use a simple web component for this
- removes web socket card
- 
image

## Pull request type

- [ ] Bugfix
- [x] Feature
- [ ] Dependency updates
- [ ] Tests
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## Do you introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the 
  [Apache Licence Version 2.0](https://github.com/terrestris/shogun/blob/main/LICENSE).
- [x] I have followed the [guidelines for contributing](https://github.com/terrestris/shogun/blob/main/CONTRIBUTING.md).
- [x] The proposed change fits to the content of the [code of conduct](https://github.com/terrestris/shogun/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added or updated tests and documentation, and the test suite passes (run `mvn test` locally).
- [x] I have added a screenshot/screencast to illustrate the visual output of my update.

### CODE REVIEW

File: shogun-boot/src/main/resources/templates/index.html [hunk 1]
```diff
@@ -42,106 +46,184 @@
...
+      // Register login/logout actions
+      document.querySelector('#login-btn').addEventListener('click', () => {
+        keycloak.login();
+      });
+      if (keycloak.authenticated) {
+        document.querySelector('#logout-btn').hidden = false;
+        document.querySelector('#login-btn').hidden = true;
+        document.querySelector('#logout-btn').addEventListener('click', () => {
+          keycloak.logout();
+        });
```
Comment (dnlkoch) [inline shogun-boot/src/main/resources/templates/index.html:160]: The event registration might be moved outside the condition (as with the login button).

File: shogun-boot/src/main/resources/templates/index.html [hunk 2]
```diff
@@ -42,106 +46,184 @@
...
+      }
+
+      const applications = await getApplications(keycloak.token);
+      const appInfos = [];
+      applications.forEach(app => {
```
Comment (dnlkoch) [inline shogun-boot/src/main/resources/templates/index.html:165]: Very minor, but we might want to switch to `.map()` instead.

**Review by dnlkoch — COMMENTED** [review]
This is awesome so far @hblitza!

**Comment by hblitza** [comment]
Thanks for the helpful review @dnlkoch. Configuration will me addressed in follow-up PR.

**Pull request merged**